libdevcore/FixedHash.h

```diff
@@ -291,6 +291,7 @@ class SecureFixedHash: private FixedHash<T>
 
 	bytesConstRef ref() const { return FixedHash<T>::ref(); }
 	byte const* data() const { return FixedHash<T>::data(); }
+	byte* data() { return FixedHash<T>::data(); }
```

Comment:
Should FixedHash be used instead of modifying SecureFixedHash to be mutable?  I suspect this was intentionally left out.